pkg/vulnerability/vulnerability.go

```diff
@@ -50,6 +52,11 @@ func (c Client) FillInfo(vulns []types.DetectedVulnerability, light bool) {
 			vulns[i].Vulnerability.Severity = severity.String()
 		} else {
 			vulns[i].Vulnerability, err = c.dbc.GetVulnerability(vulns[i].VulnerabilityID)
+			if osFound != nil { // override severity is one from vendor exists
```

Comment:
If it is vulnerabilities of libraries, we should not the severity of a vendor. We have to think about the way to display the vendor severity for os package vulnerabilities and the NVD/public database severity for library vulnerabilities.

Reply:
Roger that. With the latest changes, this is now possible:

<img width="1191" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/1254783/77489507-796fb300-6df5-11ea-9dca-cd654db4cacf.png">

Reply:
I think we don't have to store `VendorSeverity` in the result.

Reply:
Yeah currently `DetectedVulnerability ` includes `Vulnerability` which includes `VendorSeverity`. https://github.com/aquasecurity/trivy/blob/aca31dffb36328d17f6d3a3bd1184cfcd36ffc07/pkg/types/vulnerability.go#L8-L16 

Is there a way we can get rid of it in the result?

Reply:
I added source https://github.com/aquasecurity/trivy/pull/433/commits/bebd23c047d0620a1a6e6d34bb40b2fe8aa8672e and removed the VendorSeverity here https://github.com/aquasecurity/trivy/pull/433/commits/7d7d66085e388907bf5f59032b624ec849577d83